modules/nw-dns-operator-managementState.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * networking/dns-operator.adoc
+
+[id="nw-dns-operator-managementState_{context}"]
+= Changing the DNS Operator managementState
+
+DNS manages the CoreDNS component to provide a name resolution service for pods and services in the cluster. The `managementState` of the DNS Operator is set to `Managed` by default, which means that the DNS Operator is actively managing its resources. You can change it to `Unmanaged`, which means the DNS Operator is not managing its resources.
+
+The following are use cases for changing the DNS Operator `managementState`:
+
+* You are a developer and want to test a configuration change to see if it fixes an issue in CoreDNS. You can stop the DNS Operator from overwriting the fix by setting the `managementState` to `Unmanaged`.
+
+* You are a cluster administrator and have reported an issue with CoreDNS, but need to apply a workaround until the issue is fixed. You can set the `managementState` field of the DNS Operator to `Unmanaged` to apply the workaround.
+
+.Procedure
+
+* Change `managementState` DNS Operator:
++
+[source,terminal]
+----
+$ oc patch configs.dns.operator.openshift.io cluster --type merge --patch '{"spec":{"managementState":"Unmanaged"}}'
```

Comment:
I cannot find the resource `configs.dns.operator.openshift.io` and seems the command doesn't work.
```
$ oc patch configs.dns.operator.openshift.io cluster --type merge --patch '{"spec":{"managementState":"Unmanaged"}}'
error: the server doesn't have a resource type "configs"
```
@Miciah could you help confirm

Reply:
I found one resource named `configs.operator.openshift.io` has the `.spec.managementState` but it looks not for DNS operator, see
```
$ oc get configs.operator.openshift.io cluster -ojson | jq '.spec'
{
  "logLevel": "Normal",
  "managementState": "Managed",
  "operatorLogLevel": "Normal"
}
```

Reply:
According to https://github.com/openshift/api/pull/888/files, the correct command should be
```
oc patch dns.operator.openshift.io default --type merge --patch '{"spec":{"managementState":"Unmanaged"}}'
```

Reply:
Thank you! PR updated.